svg/svg.go

```diff
@@ -11,28 +11,28 @@ package svg
 func GetBullet(color string) string {
 	switch color {
 	case "":
-		return `<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" preserveAspectRatio="none" width="16" height="16"><defs><path fill="#fff" fill-opacity="0" d="M16 0H0v16h16V0z" id="a"/></defs><use xlink:href="#a"/></svg>`
```

Comment:
`preserveAspectRatio="none"` this thing is useless if an SVG does not have `viewBox` attribute:

https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/preserveAspectRatio